src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -107,23 +107,23 @@ class FormConfigBuilder implements FormConfigBuilderInterface
     /**
      * Creates an empty form configuration.
      *
-     * @param string|int               $name       The form name
+     * @param string|null              $name       The form name
      * @param string|null              $dataClass  The class of the form's data
      * @param EventDispatcherInterface $dispatcher The event dispatcher
      * @param array                    $options    The form options
      *
      * @throws InvalidArgumentException if the data class is not a valid class or if
      *                                  the name contains invalid characters
      */
-    public function __construct($name, ?string $dataClass, EventDispatcherInterface $dispatcher, array $options = [])
+    public function __construct(?string $name, ?string $dataClass, EventDispatcherInterface $dispatcher, array $options = [])
```

Comment:
`FormBuilder` extends `FormConfigBuilder`. In FormBuilder the $name was already changed in #24722 so it must be here too.